cmd/statusd/whispercfg.go

```diff
@@ -12,11 +12,13 @@ import (
 // whisperConfig creates node configuration object from flags
 func whisperConfig(nodeConfig *params.NodeConfig) (*params.NodeConfig, error) {
 	whisperConfig := nodeConfig.WhisperConfig
+
 	whisperConfig.Enabled = true
 	whisperConfig.EnableMailServer = *enableMailServer
 	whisperConfig.LightClient = *lightClient
 	whisperConfig.MinimumPoW = *minPow
 	whisperConfig.TTL = *ttl
+	whisperConfig.EnableNTPSync = *ntpSyncEnabled
```

Comment:
the same – it seems like a totally separate problem fixed here, not related to fleet configuration